package.json

```diff
@@ -170,7 +171,7 @@
     "sass-loader": "^7.3.1",
     "sequelize-cli": "^5.5.1",
     "strip-json-comments": "^3.0.1",
-    "stylelint": "^11.1.1",
+    "stylelint": "^12.0.0",
```

Comment:
Let it be 😁 
If we were really pedantic it should be done as separate PR but let's leave it now. 😉